nextcloudappstore/core/templates/base.html

```diff
@@ -9,6 +9,7 @@
     <meta name="author" content="">
     <link rel="shortcut icon" href="{% static 'assets/img/favicon.png' %}"/>
     <title>Nextcloud App Store{% block head-title %}{% endblock %}</title>
```

Comment:
In Firefox it is a pain even if tabs are not cramped, as Firefox has a max tab width of around 22 characters or so.